checks/devnet/devnet-data-start-after-1399025-with-projected-shard.json

```diff
@@ -0,0 +1,46 @@
+{
+  "network": {
+    "blockchain": "Elrond",
+    "network": "D"
+  },
+  "http_timeout": 10,
+  "max_retries": 5,
+  "retry_elapsed_time": 0,
```

Comment:
These values will be updated when merging `feat/historical` (to easier keep track of changes).